server/devpi_server/model.py

```diff
@@ -616,7 +616,7 @@ class PrivateStage(BaseStage):
         'metadata_version', 'summary', 'home_page', 'author', 'author_email',
         'maintainer', 'maintainer_email', 'license', 'description',
         'keywords', 'platform', 'classifiers', 'download_url',
-        'supported_platform', 'comment',
+        'description_content_type', 'supported_platform', 'comment',
```

Comment:
``description_content_type`` is Metadata 2.1 according to https://github.com/pypa/twine/blob/0191a22a1ea51e4ff18fd20706094e03bb1abee4/twine/package.py#L145

I would add that and ``provides_extras`` as a separate commit.

I'll review the rest later.

Reply:
Also see 
https://www.python.org/dev/peps/pep-0566/